Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="401" judge-id="250" status="SE" scoring="ACM" run-tests="1" contest-id="3202" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>e413a592-3daa-4b6b-9773-81720abaab64</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="284" real-time="299" max-memory-used="18685952" >
  <program-stats-str>{ utime=268, stime=16, ptime=284, rtime=299, maxvsz=18685952, maxrss=19476480, nvcsw=1, nivcsw=42 }</program-stats-str>
      <input size="2">4
</input>
      <output size="0"></output>
      <stderr size="300">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000401&quot;, line 5, in &lt;module&gt;
    if q%4==0 and a%100!=0 or a%400==0:
NameError: name 'a' is not defined
</stderr>
    </test>
  </tests>
</testing-report>
